changelog.md

```diff
@@ -8,4 +8,7 @@
 -   [d]—docs improvement
 
 0.1.0 (Febuary 25, 2023)
--   🎉 initial release.
\ No newline at end of file
+-   🎉 initial release.
+
+0.2.0-SNAPSHOT
```

Comment:
Most recent to the top as it is more relevant to a present user.
After that we can merge